src/sentry/replays/consumers/recording/process_recording.py

```diff
@@ -74,15 +75,18 @@ def _process_chunk(
         message: Message[KafkaPayload],
         current_transaction: Transaction,
     ) -> None:
-        cache_prefix = replay_recording_segment_cache_id(
-            project_id=message_dict["project_id"],
-            replay_id=message_dict["replay_id"],
-            segment_id=message_dict["id"],
-        )
+        with current_transaction.start_child(
+            op="replays.process_recording._process_chunk", description="process_chunk"
+        ), start_profiling(current_transaction, current_transaction.hub):
```

Comment:
I haven't tested how this call will be affected if there's other work between the start of the transaction and the start of the profile. I believe what will happen is that the start/end of the profile will no longer align with the transaction so the result might be slightly offset.